package.json

```diff
@@ -17,12 +17,12 @@
     "prettier": "^2.8.8"
   },
   "engines": {
-    "node": ">= 16.*",
+    "node": ">= 18.*",
```

Comment:
We should allow 16 still. We just won't care whether it runs or not.

Reply:
node doesn't matter anymore, because qunit-dom is only relevant in browsers

Reply:
Though considering the other comments, it could be true if it were an ember-addon.